tests/examples/source/nodejs/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM quay.io/phmartin/node:17
+# Copied base image from docker hub to quay.io  
+# skopeo copy --all docker://quay.io/odo-dev/node:19 docker://quay.io/odo-dev/node:19
```

Comment:
Thanks for adding the comment. You meant `docker.io/node:19` as the source image, right?

```suggestion
# skopeo copy --all docker://docker.io/node:19 docker://quay.io/odo-dev/node:19
```

Reply:
@rm3l  Apologies i made changes

Reply:
No problem - thanks for the changes!